### PR TITLE
Add fulltext indices

### PR DESCRIPTION
**Pull request by gbirke**
Add full text indices for name, email and city in donation and
membership tables.

This is for https://github.com/wmde/fundraising/issues/1241

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,17 @@ run style checks use `composer cs`.
...
+### Version 5.0.0 (2017-03-29)
+
+#### Breaking changes
+
+* Added full text indices for donations and memberships. This breaks backward compatibility for MySQL versions < 5.6.
+* Prefixed all indices of donations with `d_`.
+
+#### Other changes
+
+* Updated `.travis.yml` to install MySQL 5.6.
```
Comment (JeroenDeDauw) [inline README.md]: This is not something for the README as it's not relevant to people using this component

File: src/Entities/MembershipApplication.php [hunk 2]
```diff
@@ -11,7 +11,15 @@
 /**
  * @since 2.0
  *
- * @ORM\Table(name="request", indexes={@ORM\Index(name="idx_donation_id", columns={"donation_id"})})
+ * @ORM\Table(
+ *     name="request",
+ *     indexes={
+ *          @ORM\Index(name="idx_donation_id", columns={"donation_id"}),
```
Comment (JeroenDeDauw) [inline src/Entities/MembershipApplication.php]: `donation_id` huh?!

**Comment by JeroenDeDauw** [comment]
No objections from my side. README can just be changed whenever and the other things was already there

File: src/Entities/Donation.php [hunk 3]
```diff
@@ -11,7 +11,22 @@
 /**
  * @since 2.0
  *
- * @ORM\Table(name="spenden", indexes={@ORM\Index(name="email", columns={"email"}), @ORM\Index(name="name", columns={"name"}), @ORM\Index(name="ort", columns={"ort"}), @ORM\Index(name="dt_new", columns={"dt_new", "is_public"}), @ORM\Index(name="dt_exp", columns={"dt_exp", "dt_del"}), @ORM\Index(name="zahlweise", columns={"zahlweise", "dt_new"}), @ORM\Index(name="dt_gruen", columns={"dt_gruen", "dt_del"}), @ORM\Index(name="ueb_code", columns={"ueb_code"}), @ORM\Index(name="dt_backup", columns={"dt_backup"}), @ORM\Index(name="status", columns={"status", "dt_new"}), @ORM\Index(name="comment_list", columns={"is_public", "dt_del"})})
+ * @ORM\Table(
+ *     name="spenden",
+ *     indexes={
+ *     		@ORM\Index(name="d_email", columns={"email"}, flags={"fulltext"}),
+ *          @ORM\Index(name="d_name", columns={"name"}, flags={"fulltext"}),
+ *     		@ORM\Index(name="ort", columns={"ort"}, flags={"fulltext"}),
```
Comment (KaiNissen) [inline src/Entities/Donation.php]: This one does not have the `d_` prefix.

File: src/Entities/MembershipApplication.php (hunk 2 above)
Comment (KaiNissen) [inline src/Entities/MembershipApplication.php]: This was used as a foreign key in the old application. We dropped it while developing the new shiny one.

**Pull request merged**